metadata-manager/cmd/genmetadata.go

```diff
@@ -0,0 +1,234 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"log"
+	"os"
+	"time"
+
+	paperswithcode_go "github.com/codingpot/paperswithcode-go/v2"
+	models "github.com/codingpot/paperswithcode-go/v2/models"
+	"github.com/codingpot/pr12er/server/pkg/pr12er"
+	"github.com/spf13/cobra"
+	"google.golang.org/api/option"
+	"google.golang.org/api/youtube/v3"
+	"google.golang.org/protobuf/encoding/prototext"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+const (
+	defaultMappingFile  = "/server/internal/data/mapping_table.pbtxt"
+	defaultDatabaseFile = "/server/internal/data/database.pbtxt"
```

Comment:
이러면 루트에 생성 되지 않나요..?

```suggestion
	defaultMappingFile  = "server/internal/data/mapping_table.pbtxt"
	defaultDatabaseFile = "server/internal/data/database.pbtxt"
```